src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/RequestExtensionCallbackHandler.java

```diff
@@ -92,7 +93,7 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
             LocalDate.class
         );
         YesOrNo extensionAlreadyAgreed = mapper.convertValue(data.get(EXTENSION_ALREADY_AGREED), YesOrNo.class);
-        String claimNumber = "TBC";
+        String claimNumber = mapper.convertValue(data.get(LEGACY_CASE_REFERENCE), String.class);
```

Comment:
do we need to map this here?

Reply:
yes, we have to display on confirmation screen.